packages/playground/interactive-block-playground/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
CI won't be happy until this mimics tsconfig setup from other projects: three tsconfig.json files and a reference in the root tsconfig.base.json.

Reply:
done b432c707921daddc92b0ee643dfb9d067dfcf1f5 👍

Reply:
Although, I'm not sure if that's entirely correct. I've [now removed](https://github.com/WordPress/wordpress-playground/commit/0d448f81732270057dd31d1b0eb82bdee9a3f7f2) the reference in
the base tsconfig.json because I was getting a `referenced project must have
composite:true` TS error here.

The tsconfig files are just copied over from the `website` package. As far as I
can see the `website` package does not have the reference in the base tsconfig.

Reply:
> I was getting a referenced project must have composite:true TS error here.

That seems like a bug: https://github.com/nrwl/nx/issues/4508

Looks like it can be fixed by having a step called `build` in `project.json` – which is a good idea anyway if this is to benefit from all the `build`-related automations (e.g. CI build checks).

> As far as I can see the website package does not have the reference in the base tsconfig.

Ah, that was an oversight – thanks! I just fixed it in 82386e75